tests/functional/spec/features/calculated_summary/calculated_summary_test_case.js

```diff
@@ -236,86 +233,88 @@ class TestCase {
         "Total number values: 124.58",
       ];
```

Comment:
This test is failing locally for me as the texts to assert are not what im seeing in the schema 🤔

Reply:
I think the existing code didn't actually assert the texts properly

Reply:
Oh right, ya it was returning a promise as it didn't await. I must have not run it even after fixing the promise. Will sort.

Reply:
Fixed.